src/config.ts

```diff
@@ -472,6 +472,7 @@ export const linters = {
   "pylint": {
     "sourceName": "pylint",
     "command": "pylint",
+    "debounce": 500,
```

Comment:
I guess this number can be discussed. But the linter is pretty slow and resource hungry, so...